src/studio/src/designer/backend.Tests/Infrastructure/GitRepository/AltinnGitRepositoryTests.cs

```diff
@@ -70,14 +70,14 @@ public void Constructor_InvalidPathParameters_ShouldThrowException()
 
         [Theory]
         [InlineData("ttd", "apps-test", "testUser", 0)]
-        [InlineData("ttd", "ttd-datamodels", "testUser", 4)]
-        [InlineData("ttd", "hvem-er-hvem", "testUser", 8)]
+        [InlineData("ttd", "ttd-datamodels", "testUser", 0)]
+        [InlineData("ttd", "hvem-er-hvem", "testUser", 7)]
         public void GetSchemaFiles_FilesExist_ShouldReturnFiles(string org, string repository, string developer, int expectedSchemaFiles)
         {
             var repositoriesRootDirectory = TestDataHelper.GetTestDataRepositoriesRootDirectory();
             var altinnGitRepositoryFactory = new AltinnGitRepositoryFactory(repositoriesRootDirectory);
             
-            var altinnGitRepository = altinnGitRepositoryFactory.GetRepository(org, repository, developer);
+            var altinnGitRepository = altinnGitRepositoryFactory.GetAltinnGitRepository(org, repository, developer);
```

Comment:
Maybe we should reduce the excessive use of Altinn at some point?

Reply:
The use of Altinn in this case is deliberate as it indicates this is a Altinn specific Git repository as opposed to any other Git repository.

Reply:
I tough it was implicit by the name of the factory...
Where a factory with a different name would make a different kind of git repository.

Reply:
Yes, but there is also a class hierarchy involved GitRepostory -> AltinnGitRepository -> AltinnAppGitRepository and the variable names default to the class name (camleCase) which is a good thing. That said, I'm not happy with the last repository class and will probably get rid of that. First step now was to refactor a lot of logic out of the controllers. I'll continue to refactor, but need to draw the line at some point to get the code out.